.github/workflows/check-translations-and-deserialize.yml

```
@@ -69,6 +70,87 @@ jobs:
           branch-suffix: short-commit-hash
           base: develop
           delete-branch: true
+          team-reviewers: developer-enablement
+
+      - name: Checkout repository
+        if: steps.fetch-deserialize.outputs.batchesToDeserialize != 0
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Remove Completed Batches From Queue
+        if: steps.fetch-deserialize.outputs.batchesToDeserialize != 0
+        id: remove-batches
+        run: |
+          node ./scripts/actions/remove-completed-batch.js
```

Ultra super duper nitpicky:
```suggestion
          node ./scripts/actions/remove-completed-batch.js
          
```